musicbee@3.5.8698: download via aria2 failed - 404 Not Found

Hello,

When i tried to install musicbee with the command `scoop install extras/musicbee` i get:

`Installing 'musicbee' (3.5.8698) [64bit] from 'extras' bucket
Starting download with aria2 ...
Download: 03/03 23:01:58 [ERROR] CUID#8 - Download aborted. URI=https://files1.majorgeeks.com/ad9fb614274c4d3baafc0dd8dced88660177Download: Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://files1.majorgeeks.com/ad9fb614274c4d3baafc0dd8dced88660177d4Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: f3a321|ERR |       0B/s|C:/Users/kameha/scoop/cache/musicbee#3.5.8698#cf6a7ae.zip
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 3) Resource was not found
ERROR https://files1.majorgeeks.com/ad9fb614274c4d3baafc0dd8dced88660177d4b1/multimedia/MusicBeeSetup_3_5.zip
    referer=https://files1.majorgeeks.com/ad9fb614274c4d3baafc0dd8dced88660177d4b1/multimedia/
    dir=C:\Users\kameha\scoop\cache
    out=musicbee#3.5.8698#cf6a7ae.zip

ERROR & 'C:\Users\kameha\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\kameha\scoop\cache\musicbee.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/5.1 (Windows NT 10.0; Win64; x64; Desktop)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=12552 --continue --summary-interval=0 --auto-save-interval=1`

When i tried to get the file `https://files1.majorgeeks.com/ad9fb614274c4d3baafc0dd8dced88660177d4b1/multimedia/MusicBeeSetup_3_5.zip` i get a HTTP 404 Not Found

--- a/bucket/musicbee.json
+++ b/bucket/musicbee.json
@@ -6,7 +6,7 @@
         "identifier": "Freeware",
         "url": "https://musicbee.fandom.com/wiki/FAQ#Are_there_any_limitations_on_using_MusicBee.3F"
     },
-    "url": "https://files1.majorgeeks.com/ad9fb614274c4d3baafc0dd8dced88660177d4b1/multimedia/MusicBeeSetup_3_5.zip",
+    "url": "https://files1.majorgeeks.com/1864b7cce80b7e429bed0f9facbcff4eb1da3742/multimedia/MusicBeeSetup_3_5.zip",
     "hash": "be27f1149654924dddd106254493137280f3e8098ec1029435c02fdda092bdb5",
     "pre_install": [
         "(Get-ChildItem \"$dir\" 'MusicBee*.exe').FullName | Expand-7zipArchive -DestinationPath \"$dir\" -Removal",

--- a/bucket/musicbee.json
+++ b/bucket/musicbee.json
@@ -6,8 +6,8 @@
         "identifier": "Freeware",
         "url": "https://musicbee.fandom.com/wiki/FAQ#Are_there_any_limitations_on_using_MusicBee.3F"
     },
-    "url": "https://files1.majorgeeks.com/1864b7cce80b7e429bed0f9facbcff4eb1da3742/multimedia/MusicBeeSetup_3_5.zip",
-    "hash": "be27f1149654924dddd106254493137280f3e8098ec1029435c02fdda092bdb5",
+    "url": "https://files1.majorgeeks.com/771210a07ebd6b9d087f150d56412ba634fc5ffc/multimedia/MusicBeePortable_3_5.zip",
+    "hash": "a1da2ff921922fe4323f062a0032b6182b5dd000d6b5df34998614377aa3977a",
     "pre_install": [
         "(Get-ChildItem \"$dir\" 'MusicBee*.exe').FullName | Expand-7zipArchive -DestinationPath \"$dir\" -Removal",
         "Remove-Item \"$dir\\`$*\", \"$dir\\Uninst*\" -Recurse",

